tests/components/valve/test_init.py

```diff
@@ -74,17 +74,18 @@ def set_valve_position(self, position: int) -> None:
             self._attr_is_closing = True
             self._attr_is_opening = False
         self._target_valve_position = position
-        self.async_write_ha_state()
+        self.async_schedule_update_ha_state()
```

Comment:
```suggestion
        self.schedule_update_ha_state()
```

Reply:
I don't think we need to update state explicitly actually as this is a polling entity right now which will do that at the end of every service call by default.

Reply:
Maybe set `should_poll` to `False`?